specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -7088,18 +7084,22 @@
           "x-ms-format": "dfe-string",
           "type": "object"
         },
-        "keyFilePath": {
-          "description": "The full path to the .p12 key file that is used to authenticate the service account email address and can only be used on self-hosted IR. Type: string (or Expression with resultType string).",
+        "privateKey": {
+          "description": "The private key that is used to authenticate the service account email address and can only be used on self-hosted IR.",
```

Comment:
[ARMBlockingComment] If this entity is being used as a response of a get  operation, you should not be returning sensitive information. Such information must be returned as a post action

Reply:
No, this entity will not be returned in the response. The backend will handle the encryption/decryption for this sensitive information to make sure the credentials are safe. One sample response is shown below.
```json
{
    "name": "GoogleAds_ServiceAuth",
    "type": "Microsoft.DataFactory/factories/linkedservices",
    "properties": {
        "annotations": [],
        "type": "GoogleAdWords",
        "typeProperties": {
            "clientCustomerID": "123",
            "loginCustomerID": "123",
            "authenticationType": "ServiceAuthentication",
            "email": "123@456.com",
            "supportLegacyDataTypes": false,
            "googleAdsApiVersion": "v14",
            "encryptedCredential": "xxx"
        }
    }
}   
```